### PR TITLE
* [doc] add doc refresh-loading

### PR DESCRIPTION
**Pull request by miomin**
add doc refresh-loading.

### CODE REVIEW

File: doc/components/refresh-loading.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Other attributes please check out the [common attributes](../references/common-attrs.md).
+
+### Events
+
+- support `onrefresh` event  <sup class="wx-v">0.5</sup>.  If a `refresh` component is inside `scroller` / `list`, this event will be triggered when it is pull down refresh.
+- support `onloading` event .  If a `loading` component is inside `scroller` / `list`, this event will be triggered when it is pull up loadmore.
+
+### Example
+
+``` javaScript
```
Comment (luics) [inline doc/components/refresh-loading.md]: use "html"

File: doc/components/refresh-loading.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+<template>
```
Comment (luics) [inline doc/components/refresh-loading.md]: add an example in `examples/component` ?

Comment (miomin) [inline doc/components/refresh-loading.md]: done

File: doc/components/refresh-loading.md (hunk 1 above)
Comment (miomin) [inline doc/components/refresh-loading.md]: done

File: doc/components/refresh-loading.md (hunk 2 above)
Comment (miomin) [inline doc/components/refresh-loading.md]: done

**Pull request merged**